apiclient/__init__.py

```diff
@@ -22,10 +26,14 @@
     'http': http,
     'mimeparse': mimeparse,
     'model': model,
-    'sample_tools': sample_tools,
     'schema': schema,
 }
 
+try:
```

Comment:
just do `sample_tools = None` in the try..except above and you won't need to modify this at all.